srcs/runners_manager/vm_creation/CloudManager.py

```diff
@@ -70,6 +70,10 @@ def delete_vm(self, runner: Runner):
     def delete_images_from_shelved(self, name):
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def add_identifier_to_instance(self, instance_name: str, labels_webhook: dict):
+        raise NotImplementedError
+
```

Comment:
We don't need to state `webhook` in the name of this variable, right?

Following the logic on the other conversation I would go with `metadata` but just `labels` is fine as well.
```suggestion
    @abc.abstractmethod
    def add_identifier_to_instance(self, instance_name: str, metadata: dict):
        raise NotImplementedError

```